src/components/Box/Box.module.scss

```diff
@@ -1,13 +1,19 @@
 // Monday ui styles dependencies
 @import "~monday-ui-style/src/common";
+@import "~monday-ui-style/dist/mixins";
 
 // Utilities map
 @import "../../styles/utilities/utilities-settings";
 
-.box{
+.box {
   overflow: hidden;
+
+  &.scrollable {
```

Comment:
I still think the naming is confusing, it sounds like we're adding scrolling capabilities while the only thing we do is painting the scroller 🥲

Reply:
Yes we do, we have to, see line 9 lol

Reply:
I mean, we're indeed adding scrolling capabilities because the component doesn't give such right now

Reply:
But we're allowing people to add classname and they can add scroll if they need. I would even expect to get overfloe auto out-of-the box as default behavior (?)

Reply:
But it's not out-of-the-box, it's only when you use the `scrollable` prop

Reply:
Which means you explicitly want to have overflow auto, no? Otherwise the scroller mixin is useless

Reply:
Yep, I'm trying to understand if users would expect overflow auto to be the default once height is set thfough class

Reply:
Yeah I see your point, you're right